src/components/text/stories/text.stories.mdx

```diff
@@ -324,3 +324,25 @@ export const TextExamples = weight => {
 **Weight**: 400
 
 <Typeset>{TextExamples(TEXT_WEIGHT.NORMAL)}</Typeset>
+
+## Accessibility
+
+Each component has its own accessibility documentation containing a set of rules and usage examples to help implement accessible and readable text. The user can also adjust the spacing and font size to suit their own requirements using their browser settings. For this reason, the text should also be readable and not truncated when user sets line height to 1.5 times font size, spacing following paragraphs to (at least) 2 times font size, letter spacing to (at least) 0.12 times font size, word spacing to (at least) 0.16 times its size. To test that, apply following CSS rules:
+
+```css
```

Comment:
But how this affect developers decision? Line height for particular typography style is already defined.

Reply:
@clxandstuff after discussing with @katarzynatobis , I agree that this might be a good rule to follow but we need slightly more clear description with supportive examples (with screenshots), when the rule is fulfilled and what needs to be done when it's not (and why it might happen).

Reply:
@clxandstuff @coderitual As we agreed, I removed the sections about spacing and created a test to cover this comprehensively in the future.